packages/angular/package.json

```diff
@@ -72,7 +72,8 @@
     "@schematics/angular": ">= 15.0.0 < 18.0.0",
     "@angular-devkit/core": ">= 15.0.0 < 18.0.0",
     "rxjs": "^6.5.3 || ^7.5.0",
-    "esbuild": "^0.19.2"
+    "esbuild": "^0.19.2",
+    "vite": "^5.0.0"
```

Comment:
Let's not add `vite` to the plugin dependencies. We're only using a type which is only used at build time and it's stripped from the final runtime code. The actual usage of `vite` is done by the Angular builder we invoke from the `@angular-devkit/build-angular` package. That package is the one that should and does depend on it.

You can add it to the `ignoredDependencies` array in the `packages/angular/.eslintrc.json` file.